lib/topological_inventory/sync/inventory_upload/payload.rb

```diff
@@ -193,6 +193,39 @@ def sources_api_client(tenant = nil)
 
           SourcesApiClient::DefaultApi.new(api_client)
         end
+
+        def topology_inventory_api_client(tenant = nil)
```

Comment:
Should be moved together with sources api definition there: https://github.com/ManageIQ/topological_inventory-sync/blob/master/lib/topological_inventory/sync/worker.rb#L81. 
And please rename it to `topological_api_client`, this name is too long.

Reply:
I am moving methods to get instances of api clients in this PR https://github.com/ManageIQ/topological_inventory-sync/pull/47